Plugins/ExaTrkX/src/ExaTrkXPipeline.cpp

```diff
@@ -28,8 +28,9 @@ ExaTrkXPipeline::ExaTrkXPipeline(
     throw std::invalid_argument("Missing track building module");
   }
   if (m_edgeClassifiers.empty() or
```

Comment:
Not necessary for this PR, but these alternative `or` and `not` spellings are considered issues by SonarCloud I believe. We should replace them.